src/Detail/Importing/Source/DBALSource.php

```diff
@@ -2,28 +2,84 @@
 
 namespace Detail\Importing\Source;
 
-use Doctrine\DBAL;
+use Doctrine\DBAL\Driver as DBALDriver;
 
 abstract class DBALSource extends BaseSource
 {
     /**
-     * @var DBAL\Connection $connection
+     * @var DBALDriver\Connection $connection
      */
     protected $connection;
 
     /**
-     * @param DBAL\Connection $connection
+     * @var string
      */
-    public function __construct(DBAL\Connection $connection)
+    protected $dbName;
+
+
+    /**
+     * @param DBALDriver\Connection $connection
+     * @param string $dbName
+     */
+    public function __construct(DBALDriver\Connection $connection, $dbName)
     {
         $this->connection = $connection;
+        $this->dbName = $dbName;
+    }
+
+    /**
+     * @return array
+     */
+    public function fetchAll()
+    {
+        return $this->findBy();
```

Comment:
We could even make this class (`DBALSource`) non-abstract and let the repositories build the queries directly (by having access to the query builder (we'd have to bind the repositories to `DBALSource`, but that seems okay to me).

What do you think?